mult_ephem_RR-Lyr.ipynb

```diff
@@ -0,0 +1,181 @@
+   "metadata": {},
+   "source": [
+    "# RR Lyrae\n",
+    "\n",
+    "[RR Lyrae](https://en.wikipedia.org/wiki/RR_Lyrae) is the prototype of a class of variable stars with radial pulsations. The pulsation period is related to the stellar luminosity, and a second, lower frequency variation can be seen on longer timescales, known as the [Blazkho effect](https://en.wikipedia.org/wiki/Blazhko_effect). \n",
+    "\n",
+    "RR Lyrae itself was in the Kepler field, so let's load one Kepler Quarter of long-cadence observations, and inspect the periods:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c6f5e23-f9d9-4577-b012-4f138f722e9a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import astropy.units as u\n",
+    "from astropy.time import Time\n",
+    "from lightkurve import search_lightcurve\n",
+    "\n",
+    "from lcviz import LCviz\n",
+    "\n",
+    "# Download three quarters of long cadence observations from Kepler:\n",
+    "lc = search_lightcurve(\n",
+    "    target=\"RR Lyra\", \n",
+    "    mission=\"Kepler\", \n",
+    "    quarter=2, \n",
+    "    cadence=\"long\", \n",
+    "    author=\"Kepler\"\n",
+    ").download_all().stitch(\n",
+    "    # stitch together the quarters after normalizing each one:\n",
+    "    lambda x: x.normalize()\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "74d491b0-b59f-43bb-b24b-ba530d1185bb",
+   "metadata": {},
+   "source": [
+    "Now we'll start `LCviz`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96da5408-26ab-4823-b85f-4a1992c1c7dc",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "lcviz = LCviz()\n",
+    "\n",
+    "lcviz.load_data(lc)\n",
+    "\n",
+    "# we'll make use of these plugins later:\n",
+    "eph = lcviz.plugins['Ephemeris']\n",
+    "plot_options = lcviz.plugins['Plot Options']\n",
+    "freq_analysis = lcviz.plugins['Frequency Analysis']\n",
+    "freq_analysis.auto_range = False\n",
+    "freq_analysis.xunit.selected = 'period'\n",
+    "\n",
+    "lcviz.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab983eb0-6da0-45ee-a343-edfa1155f384",
+   "metadata": {},
+   "source": [
+    "Compute the peak power at periods near 13 hours using the `Frequency Analysis` plugin, and use the `Ephemeris` plugin to add a new viewer that's phase-folded at the pulsation period:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17261527-c741-4cc7-a08c-4df10a9daef6",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# compute pulsation period:\n",
+    "freq_analysis.minimum = 0.1\n",
+    "freq_analysis.maximum = 1.0\n",
+    "ls = freq_analysis.periodogram\n",
+    "period_pulsation = 1 / ls.frequency[ls.power.argmax()]\n",
+    "\n",
+    "# update Ephemeris:\n",
+    "eph.component.add_choice('pulsation')\n",
```

Comment:
this works, but might be a little simpler to avoid the extra level (and to avoid having an unused `default` ephemeris hanging around):
```suggestion
    "eph.rename_component('default', 'pulsation')\n",
```